### PR TITLE
fix: qualify E2E testing scope

### LINKED ISSUE #796: Missing API commands
## Issue

The [README](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/README.md) description of this repo says that "every API command in Cypress" is tested. This is not completely the case.

> This is an example app used to showcase [Cypress.io](https://www.cypress.io/) testing. The application uses every API command in Cypress for demonstration purposes. Additionally this example app is configured to run tests in various CI platforms. The [tests](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/cypress/e2e) are also heavily commented. For a full reference of our documentation, go to [docs.cypress.io](https://docs.cypress.io/).

### Missing API commands

The Cypress tests in [cypress-io/cypress-example-kitchensink](https://github.com/cypress-io/cypress-example-kitchensink) do not cover the following commands. They are also not described in the app on https://example.cypress.io/:

- [cy.origin()](https://docs.cypress.io/api/commands/origin)
- [cy.selectFile()](https://docs.cypress.io/api/commands/selectfile)
- [cy.shadow()](https://docs.cypress.io/api/commands/shadow)
- [cy.pause()](https://docs.cypress.io/api/commands/pause) (only makes sense to be used manually)
- [cy.session()](https://docs.cypress.io/api/commands/session)
- [cy.task()](https://docs.cypress.io/api/commands/task)

## Suggestion

1. Either add the missing commands (see above) to the Cypress tests in [cypress-io/cypress-example-kitchensink](https://github.com/cypress-io/cypress-example-kitchensink) and to the descriptions in the app on https://example.cypress.io/ OR
2. Modify the text to avoid implying that every available Cypress API command is tested.

### PR DESCRIPTION
**Pull request by MikeMcC399**
- closes #796

## Issue

The repo description does not take account of the addition of Component Testing to the previously available End-to-End Testing. `cypress-example-kitchensink` provides for E2E Testing only, not Component Testing.

The repo also claims that "every API command in Cypress" is tested, which is not completely true.

## Changes

The following locations are updated:

- [README](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/README.md)
- [package.json](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/package.json)
- [app/index.html](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/app/index.html)

1. The testing is qualified as "Cypress.io End-to-End (E2E) testing"
2. A link to [cypress-io/cypress-realworld-app](https://github.com/cypress-io/cypress-realworld-app) is added to the [README](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/README.md). ""

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Preview

[README](https://github.com/MikeMcC399/cypress-example-kitchensink/blob/testing-scope/README.md)

Locally

```shell
npm start
```

Open http://localhost:8080/

image

**Comment by MikeMcC399** [comment]
CI fails due to known issue

```text
Chrome version major is 122
Installed version of Google Chrome is 122.0.6261.94 
404
Matching Chrome Driver Version 404'd, falling back to first matching major version.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   937  100   937    0     0  14410      0 --:--:-- --:--:-- --:--:-- 14415
New ChromeDriver version to be installed: 122.0.6261.94
122.0.6261.94 will be installed
curl: (22) The requested URL returned error: 404

Exited with code exit status 22

```

**Comment by MikeMcC399** [comment]
@jennifer-shehane 

- Is it possible to re-run the failed CircleCI job in verbose mode to find out the reason for the failure also reported in https://github.com/cypress-io/circleci-orb/issues/460? The regular logs don't show:
1. which version of https://circleci.com/developer/orbs/orb/circleci/browser-tools is being used
2. what the URL is which is returning `404`

**Comment by MikeMcC399** [comment]
- See https://github.com/cypress-io/circleci-orb/issues/460#issuecomment-1974810503 for a suggestion to resolve the CircleCI issue.

**Comment by jennifer-shehane** [comment]
@MikeMcC399 Is this ready for review?

**Comment by MikeMcC399** [comment]
@jennifer-shehane 

> Is this ready for review?

Yes, I was just waiting for CI to successfully complete, which it has now done.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+This is an example app used to showcase [Cypress.io](https://www.cypress.io/) End-to-End (E2E) testing. The application demonstrates the use of most [Cypress API commands](https://on.cypress.io/api). Additionally this example app is configured to run E2E tests in various CI platforms. The [tests](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/cypress/e2e) are also heavily commented.
 
-To see the kitchen sink application, visit [example.cypress.io](https://example.cypress.io/).
+To see the kitchen sink application, and to view the [Cypress API commands](https://on.cypress.io/api) demonstrated by the app, visit [example.cypress.io](https://example.cypress.io/).
+
+For a full reference of our documentation, go to [docs.cypress.io](https://docs.cypress.io/).
+
+For an example payment application demonstrating real-world usage of Cypress.io End-to-End (E2E) testing, go to the [cypress-io/cypress-realworld-app](https://github.com/cypress-io/cypress-realworld-app) repository.
```
Comment (jennifer-shehane) [inline README.md:9]: Excellent 👍🏻

**Review by jennifer-shehane — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MikeMcC399) [reply README.md:9]: I kept the description generic as the RWA does not exercise all the API calls either.

**Pull request merged**